### DISCUSSION TITLE
get /developers error on Manager

### DISCUSSION THREAD

**Discussion by AntonOttoW** [apiman/apiman#1453] [Help] [answered]
Hi,

Just again thanks for the help thus far. I have an issue when running the get developers endpoint on manager. It returns an internal server error. I then went on and did a post developer and got a status 200, but the 'get' still throws an error afterwards.

Below are some screenshots from manager as well as the error on the logs:

![GetDevelopersRest](https://user-images.githubusercontent.com/28915136/130017239-4de1ef96-5ef9-4fbc-9647-62d40b768c78.PNG)

![DevelopersLog](https://user-images.githubusercontent.com/28915136/130017198-67676b57-b3c4-4994-b43e-a16e09962f84.PNG)

**Comment by msavy** [comment, marked as answer]
Are you using the SQL backend?  A comcontribution was made in version 2.0.0 that did not implement the JDBC/SQL backend of the developer portal functionality (only for Elasticsearch); this included the `/developers` endpoint(s). If that's what you are trying to use use, then it won't work yet.

I'm currently working on it as part of a much larger set of work, and it will land in Apiman 3.

If that is not the case, then can you please paste the stack trace in a code block rather than screenshots, as I can't see the whole thing nor analyse it easily.

Thanks

**Reply by AntonOttoW** [reply]
Hi Marc,

Okay yeah, I'm using the SQL backend... I'll then stick with the current set up then and exclude the developer portal.

Thanks

**Reply by msavy** [reply]
Keep an eye on the Apiman announcement channels for v3 lineage, which should hopefully provide everything needed 👍. A sponsor is helping fund this work.
